package.json

```diff
@@ -157,7 +157,7 @@
   "bundlesize": [
     {
       "path": "./dist/instantsearch.production.min.js",
-      "maxSize": "70.25 kB"
+      "maxSize": "71.75 kB"
```

Comment:
This is the cost of `htm` + the new code to support it. When we remove Hogan.js later on, this should significantly decrease.